backends/tfhe-cuda-backend/cuda/include/integer/integer_utilities.h

```diff
@@ -5084,6 +5084,82 @@ template <typename Torus> struct int_sub_and_propagate {
   }
 };
 
+template <typename Torus> struct int_extend_radix_with_sign_msb {
```

Comment:
For the name of this struct it should be `int_extend_radix_with_sign_msb_buffer`